src/Step.js

```diff
@@ -40,6 +57,7 @@ const Step = ({ children, name, validator, autoSkip }: Props) => {
 Step.defaultProps = {
   validator: null,
   autoSkip: false,
+  state: undefined,
```

Comment:
Any reason for `undefined` here rather than `null`? Null is a type that indicates the value is intentionally set, while `undefined` is used for unset values. I therefore suggest to use `null`.

```suggestion
  state: null,
```